demisto_sdk/commands/common/tests/base_validator_test.py

```diff
@@ -137,3 +155,107 @@ def test_check_deprecated_playbook(repo):
         base_validator = BaseValidator(ignored_errors={})
         base_validator.check_deprecated(files_path)
     assert base_validator.ignored_errors['playbook-somePlaybook.yml'] == DEPRECATED_IGNORE_ERRORS_DEFAULT_LIST
+
+
+def test_check_support_status_xsoar_file(repo, mocker):
+    """
+    Given
+    - An xsoar supported integration yml.
+
+    When
+    - Running check_support_status method.
+
+    Then
+    - Ensure the resulting ignored errors list does not include the integration file name.
+    """
+    pack = repo.create_pack('pack')
+    integration = pack.create_integration('integration')
+    meta_json = json.dumps({
+        PACK_METADATA_SUPPORT: "xsoar",
+        PACK_METADATA_CERTIFICATION: "certified"
+    })
+    mocker.patch.object(BaseValidator, 'get_meta_file_content', return_value=meta_json)
+    pack.create_json_based(name=PACKS_PACK_META_FILE_NAME, content=meta_json, prefix='')
```

Comment:
consider creating in TestSuite a pack_metadata attribute to the pack (with default values passed from the c'tor), might save these lines also for other tests

Reply:
There is one, I just did not know. I'll fix that